### PR TITLE
Add about me, testimonials and professional experience

### PR DESCRIPTION
**Pull request by krauseannelize**
@skjoldar please review in detail - added about me, testimonials and prof experience with links to Bigspool recommendation and historical data project

Note to self: Delete branch after merge (forgot to create new branch so branch name confusing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,89 @@
...
+
+- **Technical, administrative and customer support** to introduce Google Suite for membership management, updating learning material and website information, recording new members and membership fees
+- **Maintain database** ensuring members records are consistent and up to date
+
+#### ENSafrica, South Africa
+
+- **Improve security and reporting** by taking initiative to build an access-card database from scratch by coordinating with team members across different locations
+- **Leveraged automation to reduce manual effort** of a labour and time intensive administrative task assigning expenses by preparing the data to apply effective filtering
+- **Technical and administrative support** to personnel to maintain efficiency and ensuring compliance with company policies
+- **Maintain database** ensuring personnel records are consistent and up to date
```
Comment (skjoldar) [inline README.md:67]: Consider this framework:
Position Title: Technical and Administrative Coordinator

Position Summary: The Technical and Administrative Coordinator is responsible for improving security and reporting systems, leveraging automation to streamline administrative tasks, providing technical support, and maintaining accurate personnel databases. This role requires excellent organizational skills, attention to detail, and the ability to collaborate effectively with team members across different locations.

Key Responsibilities:

    Security and Reporting Enhancement: Improved security and reporting by taking the initiative to build an access-card database from scratch, coordinating with team members across different locations.

    Automation Implementation: Leveraged automation to reduce manual effort in labor-intensive administrative tasks, such as assigning expenses, by preparing data for effective filtering.

    Technical and Administrative Support: Provided technical and administrative support to personnel, maintaining efficiency and ensuring compliance with company policies.

    Database Maintenance: Ensured personnel records were consistent and up to date by maintaining an accurate database.

Qualifications:

    Proven experience in technical and administrative coordination or similar roles.

    Strong understanding of security protocols and reporting systems.

    Excellent skills in automation tools and techniques.

    Proficient in database management and data integrity maintenance.

    Strong organizational and communication skills.

Key Skills:

    Initiative and proactive problem solving

    Automation and data filtering

    Technical and administrative support

    Database management and record-keeping

    Effective collaboration and coordination

Value Proposition: By improving security and reporting, leveraging automation to enhance efficiency, and ensuring accurate database maintenance, the Technical and Administrative Coordinator contributes to the organization's operational excellence and compliance with company policies.

The content itself TBD

File: README.md [hunk 2]
```diff
@@ -1,6 +1,89 @@
...
+
+This was a large financial project to capture historical stock ledgers and particularly tickled the puzzle-solving part of my brain. I am extremely proud of what I have accomplished here and, with permission, can showcase a [summary of my work](/Other/Project-Tilburg-University.pdf). The main highlights:
+
+- **Capture, prepare and clean data** in order to digitize historical stock ledgers and to create a standardized database with unique trader and transaction identifiers
+- **Research and solve problems** such as currency conversion, accounting for the change from the Julian calendar to the Gregorian calendar, consolidating duplicate accounts, interpreting weathered and damaged pages written in ink.
+
+#### College of Animal Chiropractors & Veterinary Chiropractic Learning Centre, Canada & USA
+
+- **Technical, administrative and customer support** to introduce Google Suite for membership management, updating learning material and website information, recording new members and membership fees
+- **Maintain database** ensuring members records are consistent and up to date
```
Comment (skjoldar) [inline README.md:60]: Another example to consider:
Position Title: Membership Support Coordinator

Position Summary: The Membership Support Coordinator is responsible for providing technical, administrative, and customer support for membership management, updating learning materials and website information, and maintaining accurate member records. This role requires excellent organizational skills, technical proficiency, and strong customer service abilities.

Key Responsibilities:

    Technical, Administrative, and Customer Support: Introduced Google Suite for membership management, updated learning materials and website information, and recorded new members and membership fees.

    Database Maintenance: Ensured member records were consistent and up to date by maintaining an accurate database.

Qualifications:

    Proven experience in technical and administrative support roles.

    Strong understanding of membership management systems.

    Proficiency in using Google Suite and other relevant tools.

    Excellent organizational and communication skills.

    Strong customer service abilities.

Key Skills:

    Technical support and troubleshooting

    Administrative support and organization

    Customer service and client interaction

    Database management and record-keeping

    Proficiency in Google Suite

Value Proposition: By providing comprehensive support for membership management, updating resources, and maintaining accurate records, the Membership Support Coordinator ensures the smooth operation and continuous improvement of membership services, contributing to the organization's efficiency and member satisfaction.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,89 @@
...
+- **Product development** coordinated for over 50 product line launches and helped bring over 50,000 new product SKUs to market
+- **Business continuity** improved by creating a centralized knowledge centre in Notion across multiple business divisions
+
+#### Tilburg University, Netherlands
+
+This was a large financial project to capture historical stock ledgers and particularly tickled the puzzle-solving part of my brain. I am extremely proud of what I have accomplished here and, with permission, can showcase a [summary of my work](/Other/Project-Tilburg-University.pdf). The main highlights:
+
+- **Capture, prepare and clean data** in order to digitize historical stock ledgers and to create a standardized database with unique trader and transaction identifiers
+- **Research and solve problems** such as currency conversion, accounting for the change from the Julian calendar to the Gregorian calendar, consolidating duplicate accounts, interpreting weathered and damaged pages written in ink.
```
Comment (skjoldar) [inline README.md:55]: and also this one:
Position Title: Historical Data Analyst

Position Summary: The Historical Data Analyst is responsible for the meticulous capture, preparation, and cleaning of historical data, as well as solving complex research problems to create standardized, digitized databases. This role requires keen analytical skills, attention to detail, and the ability to interpret and digitize historical documents accurately.

Key Responsibilities:

    Data Capture and Standardization: Captured, prepared, and cleaned data to digitize historical stock ledgers, creating a standardized database with unique trader and transaction identifiers.

    Problem Solving and Research: Researched and solved problems such as currency conversion, accounting for calendar changes from Julian to Gregorian, consolidating duplicate accounts, and interpreting weathered, ink-written pages.

    In-depth Historical Research: Went beyond the initial data-capture project by conducting extensive research on each individual or entity. Utilized publicly available sources like Wikipedia to fill in gaps, clarify errors, or decode unclear text. This included tracking marital status changes, inheritances, loyalties, and affiliations.

Project Highlight: Worked on the fascinating records of the Dutch East India Company (VOC), believed to be the first portfolio of shareholders from the 1600s. This project involved digitizing documents of significant historical value, including records of wealthy individuals, families, and businesses owning shares in the VOC. The data was written in archaic language and handwriting, using the old penny/shilling/pound currency system of the time.

Qualifications:

    Proven experience in data analysis and historical research.

    Strong problem-solving skills and the ability to handle complex historical data.

    Proficiency in data digitization and standardization techniques.

    Excellent attention to detail and ability to interpret historical documents.

Key Skills:

    Data capture, preparation, and cleaning

    Historical research and problem-solving

    Standardization and digitization of data

    Interpretation of historical documents

    Analytical thinking and attention to detail

Value Proposition: By combining meticulous data analysis with a deep understanding of historical contexts, the Historical Data Analyst ensures the accurate digitization and preservation of valuable historical records, contributing to the integrity and accessibility of historical data for future research and reference.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,89 @@
...
+Since 2015, all work was performed remotely and the timelines of the various projects overlap so for brevity, I list a few highlights from this period.
+
+#### Bigspool Inc., USA
+
+This started as a small project on Upwork and very quickly developed into a full-time role with my "if I don't know, I will find out'-attitude. My role grew exponentially and responsibilities along with it. The CEO Hao Jiang wrote this [recommendation](/Other/Bigspool-Recommendation.pdf) for my time with Bigspool.
+
+- **Oversee** 7 marketplaces and 5 consumer brands attending to customer support and bi-weekly inventory fulfillment and restocking orders for over 30,000 units and $250,000
+_ **Reduce operational costs** by creating integrated forecasting reports from an inventory database I built enabling effective inventory management and cutting down costs of monthly paid subscription services
+- **Product development** coordinated for over 50 product line launches and helped bring over 50,000 new product SKUs to market
+- **Business continuity** improved by creating a centralized knowledge centre in Notion across multiple business divisions
```
Comment (skjoldar) [inline README.md:48]: and this one:
Position Title: Operations and Product Development Specialist

Overview: My journey with Bigspool started as a small project on Upwork, which quickly evolved into a full-time role. With my "if I don't know, I will find out" attitude, I embraced expanding responsibilities and significantly contributed to the company's success. CEO Hao Jiang provided a [recommendation](https://other/Bigspool-Recommendation.pdf) highlighting my impact during my tenure.

Key Achievements:

    Operational Oversight: Managed 7 marketplaces and 5 consumer brands, ensuring seamless customer support, bi-weekly inventory fulfillment, and restocking orders for over 30,000 units and $250,000 in inventory.

    Cost Reduction: Developed integrated forecasting reports from a custom-built inventory database, enabling effective inventory management and reducing costs associated with monthly paid subscription services.

    Product Development: Coordinated over 50 product line launches, bringing more than 50,000 new product SKUs to market, driving innovation and expanding the product portfolio.

    Business Continuity: Enhanced business continuity by creating a centralized knowledge center in Notion, streamlining operations across multiple business divisions.

Impact: Through strategic operational oversight, innovative product development, and efficient cost management, I played a pivotal role in Bigspool's growth and success. My ability to adapt and thrive in an evolving role ensured the company’s operational efficiency and market expansion.

**Review by skjoldar — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,6 +1,89 @@
...
+#### ENSafrica, South Africa
+
+- **Improve security and reporting** by taking initiative to build an access-card database from scratch by coordinating with team members across different locations
+- **Leveraged automation to reduce manual effort** of a labour and time intensive administrative task assigning expenses by preparing the data to apply effective filtering
+- **Technical and administrative support** to personnel to maintain efficiency and ensuring compliance with company policies
+- **Maintain database** ensuring personnel records are consistent and up to date
+
+### Legal Assistant, 2001 to 2013
+
+- **Multi-million Rand business contract** secured by my coordination of team contributions to compile a due diligence report in a new, creative and visually-appealing format I designed for a globally operating company
```
[the remark this answers is not in the document]
Comment (krauseannelize) [reply README.md:71]: Currency converted and revised with ChatGPT to focus more on the design aspect:
"Secured $1 million business contract by designing an innovative due diligence report, transforming the company's standard template with a custom-branded theme, curated artwork, and a visually striking layout for a globally operating company."

**Pull request merged**